src/app/constants/telemetry.ts

```diff
@@ -0,0 +1,31 @@
+/***********************************************************
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License
+ **********************************************************/
+export enum TELEMETRY_PAGE_NAMES {
+    HUBS = 'hub_connections',
+    CLOUD_TO_DEVICE_MESSAGE = 'cloud_to_device_message',
+    DEVICE_TELEMETRY = 'device_telemetry', // same for device vs pnp telemetry vs module telemetry
```

Comment:
I appreciate the fact that you are trying to group similar funcitonal pages together, but we would like to separate the pnp and non pnp events page, so we have an idea of the actual usage data.